proto/sedachain/wasm_storage/v1/wasm_storage.proto

```diff
@@ -28,4 +28,14 @@ enum WasmType {
       [ (gogoproto.enumvalue_customname) = "WasmTypeRelayer" ];
 }
 
+enum RequestType {
```

Comment:
Oops I should have pointed out earlier, but this is a misnomer because the wasm type is either Data Request or Overlay. I don't think we will need `RequestType` since these two types can be determined by tx type. Yes I know it's a little confusing 😂

Reply:
it is to identify the event. But I'll change back to two events.